### DISCUSSION TITLE
有没有需要人一起做AI方向科研的

### DISCUSSION THREAD

**Discussion by protoss1235** [mli/paper-reading#247] [General]
如题，本人从事AI行业的码农，但不是科研背景，想建立一下学术背景，求带

**Comment by yizhilll** [comment]
@hongbilu 可以联系我～，最近跟小伙伴一起在组织 [M-A-P research community](https://m-a-p.ai/)，有很多 NLP、Music 和 AIGC 方向的 project 想做
